tests/rptest/test_suite_quick.yml

```diff
@@ -18,7 +18,9 @@ quick:
   - tests/compacted_term_rolled_recovery_test.py
   - tests/compacted_topic_verifier_test.py
   - tests/compaction_recovery_test.py
-  - tests/wasm_identity_test.py
+  - tests/wasm_identity_test.py::WasmAllInputsToAllOutputsIdentityTest.verify_materialized_topics_test
+  - tests/wasm_failure_recovery_test.py::WasmMeshFailureRecoveryTest.verify_materialized_topics_test
+  - tests/wasm_identity_test.py::WasmRPMeshFailureRecoveryTest.verify_materialized_topics_test
```

Comment:
```
but for some reason the ducktape kafka consumer
isn't reading the expected amt of data.
```

that seems like something we should look into?

Reply:
Yeah I dedicated two solid days to this. At the end of my investigation I confirmed that the exact amount of records that was expected to be in the logs, existed. I suspect the issue may have to do with the my use of the native python kafka consumer.

Reply:
did you try a different client? there are several in ducktape

Reply:
Filed this ticket to track: https://github.com/vectorizedio/redpanda/issues/1397